docs/quickstart-as-lib.md

```diff
@@ -10,7 +10,7 @@ import (
     "log"
 
     "github.com/cayleygraph/cayley"
-    "github.com/cayleygraph/quad"
+    "github.com/cayleygraph/cayley/quad"
```

Comment:
This import is intended to be github.com/cayleygraph/quad, we have a separate repository for it.

Reply:
hmm, yes.
Basically if  you copy the example code, and try to run it with `go run .`
under go 1.13, it fails. 
And the proposed change is needed to make it work.